lib/stanford-mods/date_parsing.rb

```diff
@@ -85,6 +86,55 @@ def self.facet_string_for_century(date_str)
         end
       end
 
+      # get String sortable value for B.C. if we have  B.C. pattern
+      #  note that these values must *lexically* sort to create a chronological sort.
+      #  We know our data does not contain B.C. dates older than 999, so we can make them
+      #  lexically sort by subtracting 1000.  So we get:
+      #    -700 for 300 B.C., -750 for 250 B.C., -800 for 200 B.C., -801 for 199 B.C.
+      # @param [String] date_str String containing B.C.
+      # @return [String, nil] String sortable -ddd if B.C. in pattern; nil otherwise
+      def self.sortable_year_for_bc(date_str)
```

Comment:
I hear ya - wondering if it is less overhead / clearer to downstream class to instantiate a DateParsing object   vs.   utilizing scoped methods like 

- Stanford::Mods::DateParsing.facet_string_from_date_str(my_date_string)
- Stanford::Mods::DateParsing.sortable_string_from_date_str(my_date_string)

I may be thinking too much about java practices and solrmarc crap ... but the notion of class methods, such as there are for Date and DateTime was what I was thinking about, TBH.